### PR TITLE
added auto-generated examples

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
Ready/In Progress

## Description
refactor the script and added auto-generated examples for the commands (given a command file)

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by Itay4** [comment]
@yaakovi cool! can you provide some examples/screenshots please?

File: Scripts/script-DocumentationAutomation.yml [hunk 1]
```diff
@@ -9,30 +9,96 @@ script: |-
...
+              md_example = entry.get('HumanReadable', json.dumps(entry['Contents']))
+
+          except RuntimeError as e:
+              demisto.results("The provided example for cmd {} has failed...".format(cmd['name']))
+              demisto.results(res)
+          except Exception as e:
+              demisto.results('Error encountered in the processing of command {}, error was: {}'.format(cmd['name'], str(e)) +
+                  ". Please check your command inputs and outputs")
+      else:
+          demisto.results('did not get any example for {}. please add it manually.'.format(cmd['name']))
```
Comment (yaakovi) [inline Scripts/script-DocumentationAutomation.yml]: this needs to be -:
* triggered only if `commands` file is given.
* collected and shown as a single entry.

**Comment by avidan-demisto** [comment]
Another thing is since characters such as matching brackets, parentheses, etc, can have special meaning in markdown, if they are present in argument or parameter descriptions it might make sense to escape those characters?

**Comment by anara123** [comment]
@yaakovi 
code review notes:

bad entry id - not a file

Error from Scripts is : Script failed to run: 
Error: [Traceback (most recent call last):
   File "/tmp/pyrunner/_script_docker_python_loop.py", line 459, in <module>
    exec(code, sub_globals, sub_globals)
   File "<string>", line 294, in <module>
   File "<string>", line 256, in main
   File "<string>", line 10, in get_yaml_obj
   File "<string>", line 41, in getFilePath
   File "<string>", line 113, in __do
 ValueError: open : no such file or directory




======================
withErrors - should have default value

======================
open a bug on markdown - artifact viewer & markdown preview


=====================
removed output description
Reason
Error from Scripts is : Script failed to run: 
Error: [Traceback (most recent call last):
   File "/tmp/pyrunner/_script_docker_python_loop.py", line 459, in <module>
    exec(code, sub_globals, sub_globals)
   File "<string>", line 294, in <module>
   File "<string>", line 273, in main
   File "<string>", line 192, in generate_commands_section
   File "<string>", line 243, in generate_single_command_section
 KeyError: 'name'
] (2604) (2603)



=========================
single entry for all command example notes

=========================
add a link to documentation in content repo, in errors and script description


==========================
support Documentation process via integration name - use Demisto API
 =========================

example for command file format
- support commands_file or commands (text)

**Pull request merged**